app/assets/stylesheets/custom.scss

```diff
@@ -128,12 +127,35 @@ a {
 
 #store-name {
   color: #504d4f;
+  font-size: 1.6em;
+  float:left;
+  margin: auto;
 }
 
 .retire-btn {
   margin-bottom: 54px;
 }
 
+#complete-your-loan {
+  text-align: center;
+}
+
+#profile-title {
+  text-shadow: 0px 2px 3px #666;
+}
+
+#profile-data {
+  min-height: 20px;
```

Comment:
Properties should be ordered background-color, border, border-color, border-radius, -webkit-box-shadow, box-shadow, margin-bottom, min-height, padding